tests/README.md

```diff
@@ -59,58 +59,57 @@ Writing meaningful tests isn't difficult but requires some diligence on how to s
 
 For a short introduction on "How to write a test for Semantic MediaWiki", have a look at [this](https://www.youtube.com/watch?v=v6JRfk5ZmsI) video.
 
-### Test cases
+<pre>
+/tests/phpunit
+├─ Benchmark
+├─ Fixtures
+├─ Integration
+│	├─ ...
+│	└─ JSONScript
+├─ Structure
+└─ Unit
+	└─ ...
+</pre>
+
+- `Benchmark` contains collections of tests running benchmarks
+- `Fixtures` contains fixed data and schemata
+- `Integration` contains tests classified as testing the integration with MediaWiki, rely on an active DB connection, or connect to any other external service
+- `Structure` tests considered to verify files, structures, or some other non Semantic MediaWiki specific context
```

Comment:
I'm too dumb to understand what this means

Reply:
Tests considered examining some structural integrity and are in a strict sense no integration tests.